src/commands/diff.ts

```diff
@@ -0,0 +1,109 @@
+import { flags } from '@oclif/command';
+import * as diff from '@asyncapi/diff';
+import * as parser from '@asyncapi/parser';
+import { load } from '../models/SpecificationFile';
+import Command from '../base';
+import { ValidationError } from '../errors/validation-error';
+import { SpecificationFileNotFound } from '../errors/specification-file';
+
+export default class Diff extends Command {
+  static description = 'find diff between two asyncapi files';
+
+  static flags = {
+    help: flags.help({ char: 'h' }),
+    format: flags.string({
+      char: 'o',
+      description: 'output format',
+      default: 'json',
+    }),
+    type: flags.string({
+      char: 't',
+      description: 'the type of output',
+      default: 'all',
+    }),
+  };
+
+  static args = [
+    {
+      name: 'first-spec-file',
+      description: 'first spec path or context-name',
+      required: true,
+    },
+    {
+      name: 'second-spec-file',
+      description: 'second spec path or context-name',
+      required: true,
+    },
+  ];
+
+  async run() {
+    const { args, flags } = this.parse(Diff);
+    const firstDocumentPath = args['first-spec-file'];
+    const secondDocumentPath = args['second-spec-file'];
+
+    const outputFormat = flags['format'];
+    const outputType = flags['type'];
+
+    let firstDocument, secondDocument;
+
+    try {
+      firstDocument = await load(firstDocumentPath);
```

Comment:
awesome 👏🏻 
As you are loading spec files using the `load` function it does support loading spec files from context. so after setting appropriate contexts the command looks something like this 
```
node bin/run diff v1 v2  
```

but I think auto-loading context files does not work correctly, also I think we don't need autoloading contexts for this command anyway.

Reply:
what do you mean by autoloading context?